R/simulation-parseModule.R

```diff
@@ -66,6 +66,16 @@ setMethod(
   ),
   definition = function(filename, defineModuleElement, envir) {
 
+    if (length(filename) > 1) {
+      fileExists <- any(file.exists(filename))
+      if (fileExists) {
+        filename <- filename[file.exists(filename)]
```

Comment:
this likely also needs to select the first element that exists in case it exists in more than one of the paths

Reply:
Good point. Done in `development`